routes/index.js

```diff
@@ -50,42 +53,59 @@ router.post('/api/users', function (req, res) {
     console.log('Input field empty')
     return res.status(400).send('Input field empty')
   }
-
   // Checks for duplicate
-  const checkdup = {
-    text: 'SELECT COUNT(email) FROM users WHERE email = $1',
-    values: [email.toLowerCase()]
-  }
-  db.query(checkdup)
+  CheckDup(email)
     .then(result => {
-      if (result.rows[0].count !== 0) {
-        res.status(240)
+      if (result != 0) {
+        // console.log(result)
+        res.status(409).send({message:"exist"})
+      } else {
+
+        // Insert user data
+        const query = {
```

Comment:
[nit] Rename `const query` to `const insertUser` or something, so it's way more readable 

```db.query(insertUser)``` rather than ```db.query(query)```


 (and then, you don't need to add a comment -- implicit documentation is (usually) better than explicit!)